README.md

```diff
@@ -45,12 +45,12 @@ in other DEs, but we don't regularly test those.
 Here is a table containing the features of the app across platforms. This reflects the current
 state of latest master, not necessarily any existing release.
 
-|                               | Windows | Linux | macOS | Android |
-|-------------------------------|:-------:|:-----:|:-----:|:-------:|
-| OpenVPN                       |    ✓    |   ✓   |   ✓   |         |
-| WireGuard                     |    ✓    |   ✓   |   ✓   |    ✓    |
-| OpenVPN over Shadowsocks      |    ✓    |   ✓   |   ✓   |         |
-| Optional local network access |    ✓    |   ✓   |   ✓   |    ✓    |
+|                               | Windows | Linux | macOS | Android | iOS |
+|-------------------------------|:-------:|:-----:|:-----:|:-------:|:---:|
+| OpenVPN                       |    ✓    |   ✓   |   ✓   |         |     |
+| WireGuard                     |    ✓    |   ✓   |   ✓   |    ✓    |  ✓  |
+| OpenVPN over Shadowsocks      |    ✓    |   ✓   |   ✓   |         |     |
+| Optional local network access |    ✓    |   ✓   |   ✓   |    ✓    |  ✓  |
```

Comment:
LAN access is not really "optional" in our iOS app at the moment. LAN is always allowed currently. There are some new possibilities with iOS 14+ that might allow locking it down, but we don't implement that yet.

Could you update this checkbox to have an asterisk and below the table put something like

```
* The local network is always accessible on iOS with the current implementation
```

Watch out for markdown formatting so it does not become a bullet list.